app/src/main/java/io/github/androidacademyglobal/AppActivity.kt

```diff
@@ -22,22 +22,22 @@ class AppActivity : ComponentActivity() {
                     modifier = Modifier.fillMaxSize(),
                     color = MaterialTheme.colors.background
                 ) {
-                    Greeting(io.github.anroid_acagemy_global.core.Greeting().greeting())
+                    Greeting("Android")
```

Comment:
Were these changes necessary for Profile screen to work? :)

Reply:
Nope, I rolled back these change